### ISSUE TITLE
Cannot read property 'BANDWIDTH' of undefined

### ISSUE THREAD

**Issue by SzHeJason** [videojs/videojs-contrib-hls#1205]
![image](https://user-images.githubusercontent.com/20609396/28398586-6f7db2de-6d3a-11e7-9bd0-1b09418ccb4c.png)
![image](https://user-images.githubusercontent.com/20609396/28398599-8690e810-6d3a-11e7-884c-c0031d49ce61.png)

**Comment by zcoding** [comment]
+1

**Comment by zcoding** [comment]
video.js v5.19.2
videojs-contrib-hls v5.8.2

**Comment by fku233** [comment]
+1

 ```
 <script src="https://unpkg.com/video.js/dist/video.js"></script>
 <script src="https://unpkg.com/videojs-contrib-hls/dist/videojs-contrib-hls.js"></script>
```

**Comment by gesinger** [comment]
Thanks for the issue, however, we haven't seen it ourselves (or in our [starter template]( http://jsbin.com/vokipos/8/edit?html,console,output)). Would you be able to provide an example using our [starter template](http://jsbin.com/vokipos/8/edit?html,console,output) that shows the issue? Thank you.

**Comment by SzHeJason** [comment]
platform : MAC
browser : Chrome and firefox (ps : Safari is normal )
[Demo](http://jsbin.com/gawokunebo/edit?html,console,output)
![image](https://user-images.githubusercontent.com/20609396/28573631-cc1a7f1c-717d-11e7-89e1-34ef95356855.png)

**Comment by DullSky** [comment]
+1

**Comment by mjneil** [comment]
This bug happens when using a non master manifest as the source as the parser won't have any attribute data to add to it. we can address this either at the [parser level ](https://github.com/videojs/m3u8-parser/issues/25) or within `PlaylistLoader`

**Comment by gesinger** [comment]
@mjneil is right. We will get to https://github.com/videojs/m3u8-parser/issues/25 shortly.

**Comment by gesinger** [comment]
In progress with PR: https://github.com/videojs/videojs-contrib-hls/pull/1214

**Comment by thecotne** [comment]
@gesinger PR is merged so it will be fixed in next release right?

**Comment by gesinger** [comment]
Yup @thecotne . We'll try to cut a release soon to get this issue resolved.

**Comment by mjneil** [comment]
A fix for this has been released in v5.8.3. Please open a new issue if you continue to experience any issues.
